corehq/apps/hqcase/management/commands/reprocess_form_case_es_deletes.py

```diff
@@ -0,0 +1,90 @@
+from datetime import datetime, timedelta
+from django.core.management import BaseCommand, CommandError
+
+from corehq.apps.hqadmin.models import HistoricalPillowCheckpoint
+from pillowtop import get_all_pillow_instances
+from pillowtop.models import str_to_kafka_seq, KafkaCheckpoint
+from pillowtop.processors.elastic import ElasticProcessor
+from pillowtop.utils import get_pillow_by_name
+
+
+def get_active_pillows(pillows):
+    # return active pillows based on the heuristic 
+    #   that have their checkpoints updated in last 30 days
+    active_pillows = []
+    for pillow in pillows:
+        pillow = get_pillow_by_name(pillow)
+        last_modified = KafkaCheckpoint.objects.filter(
+            checkpoint_id=pillow.checkpoint.checkpoint_id
+        ).order_by('-last_modified')[0].last_modified
+        if last_modified > datetime.today() - timedelta(days=30):
+            active_pillows.append(pillow)
+    return active_pillows
+
+
+def get_form_es_pillows():
+    return get_active_pillows([
+        'XFormToElasticsearchPillow',
+        'xform-pillow',
+        'ReportXFormToElasticsearchPillow'
```

Comment:
Intentionally not adding custom ICDS only local pillows, since ICDS doesn't have a workflow of deleting forms/cases. Other envs don't have any custom local pillows, so using the standard pillows.